### ISSUE TITLE
Invalid zone identifier not reported as such

### ISSUE THREAD

**Issue by patrickjuchli** [moment/luxon#154]
I'd like to verify if a timezone identifier is valid. I tried using the following:

```
DateTime.utc().setZone("not_a_real_zone")
```

But this will fall back to `Settings.defaultZone` which makes sense when I'm reading `Utils.normalizeZone()`. Is this the intended behavior, though? Looking at `setZone()` I see that it wants to return `DateTime.invalid(UNSUPPORTED_ZONE)` if a zone is marked invalid. It just doesn't get the chance to do it with IANA timezones.

If this is intended, is there some other way to verify the timezone using Luxon?

**Comment by icambron** [comment]
Since `setZone()` takes lots of different kinds of arguments, it first says "is this supposed to be an IANA zone? If so, is it valid?" Whereas for your string it doesn't try to interpret it as an IANA zone at all (no slashes, basically), so it doesn't find out that it's not.

```js
// this works:
DateTime.utc().setZone("Vloobs/Blarks").invalidReason //=> "unsupported zone"
DateTime.utc().setZone("justBlarks").invalidReason // => null
```

I think I was just being lazy about not providing a clean way for `normalizeZone` to signal back nothingness.

I'd be willing to take two changes:

 1. Change the defaulting to invalid zone instead of default.  I suspect adding an InvalidZone class and returning a singleton instance of it is probably the easiest thing to do here, but I haven't thought all that through.
 2. Provide `Util.isValidIANAZone(s)` (it would just wrap the `isValid` function in IANAZone.js, which would need to be promoted to a class method)

**Comment by patrickjuchli** [comment]
I agree with both points. A zone called "justBlarks" shouldn't return a valid zone. I might have time to take this on in the coming days, but not sure.

**Comment by icambron** [comment]
I fixed item 1 in 80dba8f. Added 2 in 8a92d80. Will release shortly.
